### ISSUE TITLE
Collection dropdown in Submission UI should not appear if disabled in backend configuration

### ISSUE THREAD

**Issue by tdonohue** [DSpace/dspace-angular#673]
As discussed in this week's DSpace 7 meeting and in https://wiki.lyrasis.org/display/DSPACE/Collection+dropdown+in+submission, we need to support the ability to **disable the Collection dropdown** in the Submission UI.  This change is necessary as not all institutions may be willing to accept the side effects of changing a Collection after already starting a submission (see wiki page for details on those side effects)

As proposed in the above wiki page, the idea is to not display this dropdown if the "collection" step is _not included_ in the submission configuration.  This would, in effect, revert to DSpace 6-like behavior, where the Collection must be specified *prior to starting the submission* and cannot be changed after the fact. 

This change is dependent on #671, as that separate ticket defines how we will specify the Collection (on the MyDSpace page) prior to starting the submission.
